harvardkey_cas/backends.py

```diff
@@ -30,30 +30,33 @@ def authenticate(self, request, ticket, service):
         """
 
         client = get_cas_client(service_url=service)
+        logger.debug(f'CAS client type: {type(client)}')
```

Comment:
What do we expect this to show? The class name of the object returned by `get_cas_client()`? Does that indicate e.g. a particular configuration or version? Just curious what this will be useful for.

Reply:
Yeah -- there are different versions of the CAS client (`CASClientV1`, `CASClientV2`, `CASClientV3`, `CASClientWithSAMLV1 `) that could be returned. This will just help us know exactly which one we're working with.